frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -77,6 +98,23 @@ export const FieldRowContainer = ({
     [handleFieldClick],
   )
 
+  const handleDuplicateClick = useCallback(() => {
+    duplicateFieldMutation.mutate(field._id)
```

Comment:
should we add a guard here to only allow duplicate if the field is in the creation stage?

Reply:
7f67559a